sentry-android-core/src/test/java/io/sentry/android/core/SentryAndroidOptionsTest.kt

```diff
@@ -26,12 +26,12 @@ class SentryAndroidOptionsTest {
         assertEquals(BuildConfig.VERSION_NAME, sdkVersion.version)
 
         assertTrue(sdkVersion.packages!!.any {
-            it.name == "maven:sentry-android-core"
+            it.name == "maven:sentry-android-core" &&
             it.version == BuildConfig.VERSION_NAME
         })
 
         assertTrue(sdkVersion.packages!!.any {
```

Comment:
Honestly I'd have a single assert per condition here so:
```suggestion
        assertTrue(sdkVersion.packages!!.any { it.name == "" });
        assertTrue(sdkVersion.packages!!.any { it.version == "" });
```

Reply:
If we change it, it would be possible to have a one package matching first assertion and another (or the same) package matching second one, right? This test is meant to verify if there is a single package that meets both criteria.

Reply:
yes, agree with @maciejwalkowiak